course.py

```diff
@@ -248,3 +248,44 @@ def add_duplicate_course():
         db.session.rollback()
 
     return redirect(url_for('course.course_info', course_id=course_id, current_year=year))
+
+
+@course_bp.route('/evaluations')
+@login_required
+def evaluations():
+    user_id = request.args.get('user_id')
+    current_year = request.args.get('current_year')
+    courses = db.session.query(Course).filter_by(year=current_year).all()
+
+    return render_template('evaluations.html', courses=courses, current_year=current_year, user_id=user_id)
+
+
+@course_bp.route('/create_evaluations', methods=['POST'])
+@login_required
+def create_evaluation():
+    form = request.form
+    if not form:
+        return make_response("Problem with form request", 500)
+
+    course_id = request.form['course_id']
+    current_year = request.args.get('current_year')
+    user_id = request.args.get('user_id')
+    tasks = request.form.getlist('tasks[]')
+    evaluation_hour = request.form['evaluation_hour']
+    workload = request.form['workload']
+    comment = request.form['comment']
+    second_course = True if request.form.get('second_course') == 'Yes' else False
+
+    tasks_str = ', '.join(tasks)
```

Comment:
You may want something more robust and keep a list in a JSON (or equivalent) format. This way the code won't break even if we decide to add a task including a comma.